Resources/Prototypes/Entities/Mobs/Customization/Markings/undergarments.yml

```
@@ -92,132 +92,135 @@
+#   id: UndergarmentBottomSatinReptilian
+#   bodyPart: UndergarmentBottom
+#   markingCategory: UndergarmentBottom
+#   speciesRestriction: [Reptilian]
+#   coloring:
+#     default:
+#       type: null
+#       fallbackColor: '#FFFFFF'
+#   sprites:
+#   - sprite: Mobs/Customization/undergarments.rsi
+#     state: satin_reptilian
+# Exodus-No-Need-For-Underwear-For-Race-End
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)